submit.yml.erb

```diff
@@ -4,4 +4,4 @@ batch_connect:
 script:
   native:
     resources:
-      nodes: "1:ppn=2:owens"
+      nodes: "1:ppn=2"
```

Comment:
Every adapter has it's own expectations about what's in `script.native`. I don't believe anything is necessary here for the lha.